src/extensions/clinvar/tables/VariationArchiveVariation.json

```diff
@@ -44,6 +44,17 @@
         }
       ]
     },
+    {
+      "name": "parent_ids",
+      "datatype": "integer",
```

Comment:
Same question as above: if the idea is that we'll generate a numeric ID here, can we find a combination of other columns to use instead?